src/types.jl

```diff
@@ -319,7 +319,7 @@ end
 
 function GlyphExtent(font, char)
     extent = get_extent(font, char)
-    ink_bb = FreeTypeAbstraction.inkboundingbox(extent)
+    ink_bb = FreeTypeAbstraction.height_insensitive_boundingbox(extent, font)
```

Comment:
I think this makes bounding boxes less accurate. Why not undo my changes instead? (use character height instead of font height)

Reply:
it is that way on purpose. text should not be bounding-boxed tightly around the glyphs, but given the font's ascenders and descenders. unless there is a specific purpose for having a tighltly bounded glyph, but normal text is not that

Reply:
the question is just if LaTeX should use something else because text is positioned all over the place there. I don't know how real LaTeX handles equation bounding boxes

Reply:
ah ok I guess I misunderstood, the ink_bb should of course be the inkboundingbox, but that doesn't mean we have to use it to compute text boundingboxes. it's still useful to have in that struct

Reply:
Another direction I thought about going with my changes was to use something like `overallheight = max(font_height, max_height_of_line)` for latex. I guess that would be the way if we want to keep text aligned based on the largest character/font rather than individual characters.

Maybe we should also just have both styles for alignments and bounding boxes. I.e. one based on characters and one based on the (if necessary padded) font (height)?

Reply:
ideally the font ascender / descenders shouldn't be that far off from normal strings that have the occasional f or g. Just some fonts define those metrics in a peculiar way (case in point, the need for TeX Gyre Heros Makie). So only if you were to do something where absolute precision of boundingboxes of strings was necessary would you need those other styles. I think in all other situations the non-jitteryness is preferable and seems to be what other text-based tools or browsers etc are doing.